Lib/test/test_support.py

```diff
@@ -794,6 +797,27 @@ def test_linked_to_musl(self):
             for v in linked:
                 self.assertIsInstance(v, int)
 
+    def test_threadsafe_environmentvarguard(self):
```

Comment:
This test should be executed more than once. So I would suggest doing something like that:

```py
def test_threadsafe_environmentvarguard(self):
    for _ in range(100):
        self._test_threadsafe_environmentvarguard()

def _test_threadsafe_environmentvarguard(self):
    ...
```